pinning/hardware/netio.m4

```diff
@@ -1,5 +1,7 @@
-/* add support for RS485 */
-pin(RS485TE_USART0, PD2, OUTPUT)
+ifdef(`conf_MODBUS', `dnl
+  /* add support for RS485 */
+  pin(RS485TE_USART0, PC0, OUTPUT)
```

Comment:
Uh, PC0 is I2C SCL - not a very good choice. But anyway, users have to adapt this to their own hardware.